app/controllers/state_file/questions/return_status_controller.rb

```diff
@@ -8,12 +8,16 @@ def edit
         @submission_to_show = current_intake.latest_submission
         @return_status = return_status
         @error = submission_error
+
+        @department_of_taxation = StateFile::StateInformationService.department_of_taxation(current_state_code)
+        @tax_refund_website_name = @department_of_taxation.split(" ").map(&:first).join.upcase
```

Comment:
I'm concerned here about just taking the initials of the `department_of_taxation` and using them here - but it does seem to be working out. It doesn't feel right to call it the `tax_refund_website_name` though, it would be more clear to say `@department_of_taxation_initials` since the website name could be something else